### PR TITLE
Fix/675 pipedream cap sync output

### PR DESCRIPTION
**Pull request by daveomri**
## WHY

Large synchronous Actor OUTPUT could exceed Pipedream's step-output limit, failing the step after the Actor already ran and was charged. Closes [675](https://github.com/apify/apify-integrations-backend/issues/675).

## What changed

- **`run-actor.mjs`**: added `MAX_OUTPUT_BYTES` (256 KiB cap), `outputByteSize()`, and `capOutputRecord()`. Oversized output is replaced with a reference object pointing to the full KV record. `$summary` notes when output was capped. Sync path now uses `apify.getKVSRecord()` per convention. Version `0.0.7 → 0.0.8`.
- **`run-actor.test.mjs`**: 14 unit tests covering byte-sizing and capping logic, including boundary cases.

## Testing

- `eslint` clean on both files.
- 14 unit tests pass (run via local ESM config due to pre-existing `jest.config.js` issue).
- Live check: sync Actor with large OUTPUT returns the reference object instead of an oversized payload.

### Previous
obrazek

### Updated
obrazek

### CODE REVIEW

File: components/apify/actions/run-actor/run-actor.mjs [hunk 1]
```diff
@@ -365,20 +397,20 @@ export default {
...
-          ._client()
-          .keyValueStore(run.defaultKeyValueStoreId)
-          .getRecord(outputRecordKey);
-
-        output = record?.value;
+        const record = await apify.getKVSRecord(run.defaultKeyValueStoreId, outputRecordKey);
+        output = await this.capOutputRecord(record, run.defaultKeyValueStoreId, outputRecordKey);
       }
 
+      const capped = output?.truncated === true;
```
Comment (JanHranicky) [inline components/apify/actions/run-actor/run-actor.mjs]: There might be a potential edge-case we could run into here. If an Actor run would return `truncated` as its own field (scrapper finished early or something) this would resolve to `true` and would show a false message. 

Reading the value from `capOutputRecord` should do the trick

**Review by JanHranicky — CHANGES_REQUESTED** [review]
Found potential edge-case

**Review by drobnikj — APPROVED** [review]
Agree with Honza note, otherwise fine.
